package.json

```diff
@@ -29,7 +30,7 @@
     "typescript": "^4.9.3"
   },
   "scripts": {
-    "install-deps": "pnpm install --frozen-lockfile --ignore-scripts --prefer-offline",
+    "install-deps": "pnpm i --ignore-scripts --prefer-offline",
```

Comment:
why removing this? this was added mainly CI doesn't update pnpm-lock file and use what is committed

Reply:
Ok we need to fix that before